spec/firebase_app_distribution_action_spec.rb

```diff
@@ -27,6 +27,87 @@
     Faraday.default_connection = nil
   end
 
+  describe '#get_value_from_value_or_file' do
+    it 'should return the release notes string' do
```

Comment:
A few things:

- This method doesn't actually have to do with release notes, so the description shouldn't mention them. Instead it's just concerned with values and files in general.
- You can remove the "should" phrasing, i.e: "should return..." -> "returns..."
- The description should mention the specific case under test, in this case it's that the method returns the value *"when the value is defined"*

Please make these changes throughout.